apis/core/lib/handlers/dataset.ts

```diff
@@ -42,7 +42,7 @@ export const loadCubes: Enrichment = async (req, dataset) => {
     return
   }
 
-  dataset.any().has(rdf.type, cube.Cube).forEach(cube => {
+  clownface(dataset).any().has(rdf.type, cube.Cube).forEach(cube => {
```

Comment:
Necessary because `hydra-box` still uses `0.x` of clownface which does not have `.any()`

cue zazuko/hydra-box#92